docs/install/index.md

```diff
@@ -479,9 +479,7 @@ You can use the Maven packages defined in the following dependency to include MX
 <dependency>
     <groupId>org.apache.mxnet</groupId>
     <artifactId>mxnet-full_2.11-linux-x86_64-gpu</artifactId>
-    <scope>system</scope>
-    <version>1.4.0</version>
-    <systemPath>/system/path/to/jar/mxnet-full_2.11-linux-x86_64-gpu-1.4.0-SNAPSHOT.jar</systemPath>
+    <version>LATEST</version>
```

Comment:
LATEST only works for maven2, it doesn't work for maven 3.
It's better to use: https://maven.apache.org/pom.html#Dependency_Version_Requirement_Specification

Reply:
Yes, use something like [1.4.0, )

Reply:
@frankfliu Addressed version info in latest commit